From 0bef3afae981ee00cac329b72f06f9b2d3385b69 Mon Sep 17 00:00:00 2001
From: Zhong Yichong <zyc@rock-chips.com>
Date: Fri, 9 Feb 2018 16:20:16 +0800
Subject: [PATCH] camera: rockchip: camsys_drv: disable cru reset of rk3326 isp
 now

Because it will cause CPU hangs on, so disable the reset temporarily.

Change-Id: I612be4b47145fd1ebf8d8e5d44270f26151768fa
Signed-off-by: Zhong Yichong <zyc@rock-chips.com>
---
 drivers/media/video/rk_camsys/camsys_soc_rk3326.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/media/video/rk_camsys/camsys_soc_rk3326.c b/drivers/media/video/rk_camsys/camsys_soc_rk3326.c
index c65e855ce468..f3a18326eb4e 100644
--- a/drivers/media/video/rk_camsys/camsys_soc_rk3326.c
+++ b/drivers/media/video/rk_camsys/camsys_soc_rk3326.c
@@ -194,6 +194,7 @@ int camsys_rk3326_cfg
 	}
 
 	case Isp_SoftRst: {/* ddl@rock-chips.com: v0.d.0 */
+#if 0
 		unsigned long reset;
 
 		reset = (unsigned long)cfg_para;
@@ -206,8 +207,10 @@ int camsys_rk3326_cfg
 			VI_IRCL));
 			camsys_trace(2, "Isp self soft rst: %ld", reset);
 			break;
+#else
+		break;
+#endif
 		}
-
 	default:
 	{
 		camsys_warn("cfg_cmd: 0x%x isn't support", cfg_cmd);
-- 
2.35.3

